test/specs/token-categorizer.js

```diff
@@ -34,7 +34,7 @@ describe('token-categorizer', function() {
         });
 
         it('should limit input to open parentheses', function() {
-            var nonParen = sharedFile.getPrevToken(sharedFile.getLastToken());
+            var nonParen = sharedFile.getPrevToken(sharedFile.getLastToken({includeComments: true}));
```

Comment:
I needed `getFirstToken` to ignore `Whitespace`, but just silently having this as a default behaviour seemed not so good to me. So I changed `getFirstToken` and `getLastToken` signatures to match `getPrev/NextToken` signatures.